plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/EditorConfigTests.kt

```diff
@@ -147,6 +147,26 @@ class EditorConfigTests : AbstractPluginTest() {
         }
     }
 
+    @DisplayName("Check task should rerun if additionalEditorconfig property changes")
+    @CommonTest
+    fun checkRerunOnAdditionalEditorconfigPropertyChange(gradleVersion: GradleVersion) {
```

Comment:
I made sure that the test fails if `val additionalEditorconfig` is `@Internal` before changing it to `@Input`